ERDDAP_timeseries_explorer.ipynb

```diff
@@ -266,7 +216,7 @@
    },
    "outputs": [],
    "source": [
-    "e = ERDDAP(server=server, protocol='tabledap')"
+    "e = ERDDAP(server=servers[server_name].get('server_addr'), protocol='tabledap')"
```

Comment:
Use two lines for more readability.
server = servers[server_name].get('server_addr')

e = ERDDAP(server=server, protocol='tabledap')